system/TestBox.cfc

```diff
@@ -402,17 +402,23 @@ component accessors="true"{
 			arguments.callbacks.onBundleStart( target, testResults );
 		}
 
-		// Discover type?
-		if( structKeyExists( target, "run" ) ){
-			// Run via BDD Style
-			new testbox.system.runners.BDDRunner( options=variables.options, testbox=this )
-				.run( target, arguments.testResults, arguments.callbacks );
-		}
-		else{
-			// Run via xUnit Style
-			new testbox.system.runners.UnitRunner( options=variables.options,testbox=this )
-				.run( target, arguments.testResults, arguments.callbacks );
-		}
+		try {		
+		
+			// Discover type?
+			if( structKeyExists( target, "run" ) ){
+				// Run via BDD Style
+				new testbox.system.runners.BDDRunner( options=variables.options, testbox=this )
+					.run( target, arguments.testResults, arguments.callbacks );
+			}
+			else{
+				// Run via xUnit Style
+				new testbox.system.runners.UnitRunner( options=variables.options,testbox=this )
+					.run( target, arguments.testResults, arguments.callbacks );
+			}	
+		}
+		catch (any ex) {
+			throw(message = "Error executing bundle - #arguments.bundlePath#  message: #ex.message#", detail = ex.stackTrace);
```

Comment:
Do you mind adding the `#ex.detail#` as well alongside the message?
Also, do you mind updating the white spaces to match our syntax guidelines: https://github.com/Ortus-Solutions/coding-standards.

```
try{

} catch (Any ex ){
    throw(
        message = "",
        detail       = ""
    );
}